reconcile/entries.go

```diff
@@ -205,47 +206,44 @@ func Entries(ctx context.Context, logger kitlog.Logger, cl EntriesClient, catalo
 				continue // will have been created above
 			}
 
-			entry, alreadyExists := entriesByExternalID[model.ExternalID]
-			if alreadyExists {
-				// If we found the entry in the list of all entries, then we need to diff it and
-				// update as appropriate.
-				if entry != nil {
-					isSame :=
-						entry.Name == model.Name &&
-							reflect.DeepEqual(entry.Aliases, model.Aliases)
-
-					currentBindings := map[string]client.CatalogAttributeBindingPayloadV2{}
-					for attributeID, value := range entry.AttributeValues {
-						current := client.CatalogAttributeBindingPayloadV2{}
-						// Our API behaves strangely with empty arrays, and will omit them. This patch
-						// ensures the array is present so our comparison doesn't trigger falsly.
-						if value.ArrayValue == nil && value.Value == nil {
-							value.ArrayValue = lo.ToPtr([]client.CatalogAttributeValueV2{})
-						}
+			// If we found the entry in the list of all entries, then we need to diff it and
```

Comment:
most of this diff is me removing an unecessary if, view the diff without whitespace to avoid going mad